modules/nf-core/biscuit/bsconv/main.nf

```diff
@@ -1,19 +1,17 @@
 process BISCUIT_BSCONV {
     tag "$meta.id"
-    label 'process_long'
-
+    label 'process_medium'
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/biscuit:1.1.0.20220707--he272189_1':
-        'biocontainers/biscuit:1.1.0.20220707--he272189_1' }"
+        'https://depot.galaxyproject.org/singularity/mulled-v2-d94f582b04a3edcede1215189c0d881506640fd9:6519548ea4f3d6a526c78ad0350c58f867f28574-0':
+        'biocontainers/mulled-v2-d94f582b04a3edcede1215189c0d881506640fd9:6519548ea4f3d6a526c78ad0350c58f867f28574-0' }"
 
     input:
-    tuple val(meta), path(bam), path(bai)
-    path(index)
+    tuple val(meta), path(bam)
```

Comment:
Need to have .bai input; if the -g option is specified for bsconv, it will fail without a bam index